tests/user_examples/extra_src/shared/general_continuum/continuum_dynamics/continuum_dynamics_complex.hpp

```diff
@@ -97,16 +97,42 @@ void BaseShearStressRelaxation2ndHalfWithWall<BaseShearStressRelaxation2ndHalfTy
     this->velocity_gradient_[index_i] += velocity_gradient;
 }
 
+//=================================================================================================//
+//============================Plastic:BaseStressRelaxationWithWall=================================//
+//=================================================================================================//
```

Comment:
This style of comments were not used in the previous code, remove for consistency.